lib/Test/Selenium/Remote/Driver.pm

```diff
@@ -90,15 +90,15 @@ sub BUILD {
 
 =head1 NAME
 
-Test::Selenium::Remote::Driver
+Test::Selenium::Remote::Driver - add testing methods to L<Selenium::Remote::Driver>
 
 =head1 DESCRIPTION
 
-A subclass of L<Selenium::Remote::Driver>.  which provides useful testing
-functions.
+A subclass of L<Selenium::Remote::Driver> which provides useful testing
+methods.
 
 This is an I<experimental> addition to the Selenium::Remote::Driver
-distribution, and some interfaces may change.
+distribution. Some interfaces may change.
```

Comment:
It can probably be said with confidence that this interface is no longer experimental;

There is enough code extant that I'm already afraid to change anything in here.

Reply:
Yes, that's right.